lib/shovel.js

```diff
@@ -117,8 +106,7 @@ function setupFiles(opts, strategies) {
     if (opts.files) {
         if (strategies.files) {
             strategies.files();
-        }
-        else {
+        } else {
```

Comment:
I'm more of a 
```
if () {
}
else {
}
```
kind of guy (easier to scan since all keywords are on the same vertical line) but we can go with this for now ;)